app/src/main/java/org/wikipedia/feed/FeedContentType.kt

```diff
@@ -27,9 +27,9 @@ enum class FeedContentType(private val code: Int,
             return if (isEnabled) AggregatedFeedContentClient.FeaturedArticle(aggregatedClient) else null
         }
     },
-    TRENDING_ARTICLES(3, R.string.view_top_read_card_title, R.string.feed_item_type_trending, true) {
```

Comment:
If we are renaming, should we be. more accurate and rename to "TOP_READ_ARTICCLES" per what we display?

Reply:
You are correct! I did further rename on related classes and package and it all makes sense now. Thanks!